website/versioned_docs/version-v3.14.x/sync.md

```diff
@@ -35,8 +35,59 @@ You can install this config with the following command:
 ```sh
 kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/demo/basic/sync.yaml
 ```
-
-Once data is synced into OPA, rules can access the cached data under the `data.inventory` document.
+Once data is synced, ConstraintTemplates can access the cached data under the `data.inventory` document. ConstraintTemplates must specify the Group/Version/Kinds that they will need in a `metadata.gatekeeper.sh/requires-sync-data` annotation. This is specified in a format like this:
```

Comment:
Thanks for the PR!

nit: please undo changes for 3.14 and 3.13 docs since 3.15 is when the SyncSet feature was introduced.